markdown/bedrock.css

```diff
@@ -26,7 +26,8 @@ html{
 	background:#141414;
 	font-family:'Open Sans', sans-serif;
 	-webkit-text-size-adjust:100%;
-	-ms-text-size-adjust:100%
+	-ms-text-size-adjust:100%;
+	padding: 1em
```

Comment:
Double check the output of this, adding a padding on all sides is generally dangerous, and `1em` (by default) is usually alot :)

Reply:
Updated to `padding-bottom: 3px`.  Let me know if that's better.